lib/containers/containerd_nerdctl.pm

```diff
@@ -11,15 +11,22 @@ use Mojo::Base 'containers::engine';
 use testapi;
 use containers::common 'install_containerd_when_needed';
 use containers::utils 'registry_url';
+use Utils::Architectures;
 has runtime => 'nerdctl';
 
 sub init {
     install_containerd_when_needed();
 
     # The nerdctl validation test suite is a plug-in required for this test and needs to be installed from an external source.
     my $version = get_var('CONTAINERS_NERDCTL_VERSION', '0.16.1');
-    my $url = "https://github.com/containerd/nerdctl/releases/download/v$version/nerdctl-$version-linux-amd64.tar.gz";
-    my $filename = "/tmp/nerdctl-$version-linux-amd64.tar.gz";
+    my $arch = 'amd64';
+    if (is_aarch64) {
+        $arch = 'arm64';
+    } elsif (!is_x86_64) {
+        die 'Architecture ' . get_var('ARCH') . ' is not supported';
+    }
```

Comment:
```
my $arch = 'arm64' if (is_x86_64 or is_aarch64);
die 'Architecture ' . get_required_var('ARCH') . ' is not supported' unless $arch
```

Reply:
Can't. It should be `amd64` for x86... not `arm64` :)